xrpl/asyncio/transaction/reliable_submission.py

```diff
@@ -89,6 +90,8 @@ async def send_reliable_submission(
     transaction_hash = transaction.get_hash()
     submit_response = await submit_transaction(transaction, client)
     prelim_result = submit_response.result["engine_result"]
+    if prelim_result != "tesSUCCESS":
+        raise Exception(submit_response.result["engine_result_message"])
```

Comment:
Error messages can also occur in other places in the response (this is a processing example: https://github.com/XRPLF/xrpl-py/blob/7572e939e271fa838057f6d2cc529200379af9ec/xrpl/asyncio/clients/exceptions.py#L19)